data/table_files/grib1_cptec_254_64.txt

```diff
@@ -161,24 +161,24 @@ GRIB1
 159 254 46 -1 "tppu" "TROPOPAUSE U-WIND COMPONENT" "m/s"
 160 254 46 -1 "tppv" "TROPOPAUSE v-WIND COMPONENT" "m/s"
 161 254 46 -1 "var161" "undefined" ""
-162 254 46 -1 "gvdu" "GRAVITY WAVE DRAG DU/DT" "m/s2"
-163 254 46 -1 "gvdv" "GRAVITY WAVE DRAG DV/DT" "m/s2"
+162 254 46 -1 "gvdu" "GRAVITY WAVE DRAG DU/DT" "m/s^2"
+163 254 46 -1 "gvdv" "GRAVITY WAVE DRAG DV/DT" "m/s^2"
 164 254 46 -1 "gvus" "GRAVITY WAVE DRAG SFC ZONAL STRESS " "Pa"
 165 254 46 -1 "gvvs" "GRAVITY WAVE DRAG SFC MERIDIONAL STRESS" "Pa"
 166 254 46 -1 "var166" "undefined" ""
 167 254 46 -1 "dvsh" "DIVERGENCE OF SPECIFIC HUMIDITY" "1/s"
 168 254 46 -1 "hmfc" "HORIZ. MOISTURE FLUX CONV. " "1/s"
-169 254 46 -1 "vmfl" "VERT. INTEGRATED MOISTURE FLUX CONV." "kg/(m2*s)"
-170 254 46 -1 "vadv" "VERTICAL MOISTURE ADVECTION " "kg/(kg*s)"
-171 254 46 -1 "nhcm" "NEG. HUM. CORR. MOISTURE SOURCE" "kg/(kg*s)"
+169 254 46 -1 "vmfl" "VERT. INTEGRATED MOISTURE FLUX CONV." "kg/m^2/s"
+170 254 46 -1 "vadv" "VERTICAL MOISTURE ADVECTION " "kg/kg/s"
```

Comment:
Seconds are usually on the bottom of the fraction for moisture advection.  I think this should say kg/kg*s...